tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/transaction/TransactionStmtTest.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Test cases for flow validation in TransactionStatement.
  */
-
+@Test(groups = {"disableOnOldParser"})
```

Comment:
Any reason for avoiding transaction statement test for old parser?

Reply:
A compilation error(`invalid token 'transactional'`) is given for `transactional` keyword in the old parser.